modules/op-using-pipelines-as-code-with-gitlab.adoc

```diff
@@ -105,16 +105,16 @@ metadata:
   name: my-repo
   namespace: target-namespace
 spec:
-  url: "https://gitlab.com/owner/repo" <1>
+  url: "https://gitlab.com/owner/repo" # See comment on spec.git_provider.url
```

Comment:
that's a wrong comment, suggested changes:

```suggestion
  url: "https://gitlab.com/owner/repo" # The Repository URL 
```

Reply:
Addressed. Thanks